README.md

```diff
@@ -216,7 +215,7 @@ There are 2 ways to test your changes with modification:
 
 Whenever a new api is added or a new field is added to the CRD, please make sure to run the command:
   ```commandline
-  make api-docs 
+  make api-docs
```

Comment:
unrelated whitespace change

Reply:
@ykaliuta Should these unrelated changes be reverted back?

Reply:
it's not mandatory in the project, so up to you.

Reply:
I think I'll let these be, these a unwanted spaces for better readability and creating a separate PR would be stretch.